services/backend/src/config.py

```diff
@@ -8,7 +8,7 @@ class BaseConfig:
     BCRYPT_LOG_ROUNDS = 13
     ACCESS_TOKEN_EXPIRATION = 900  # 15 minutes
     REFRESH_TOKEN_EXPIRATION = 2592000  # 30 days
-    NUMBER_OF_CONTENT_IN_ANN = 100 # UPDATE THIS WHEN DEVELOPING ANN
```

Comment:
please remove, this is for your own development only

Reply:
Thank you professor for pointing out. Will this number be increased when it is served in production?